src/style.css

```diff
@@ -1 +1,66 @@
-/* Styles go here */
+html {
+  font-family: "Roboto", sans-serif;
+  font-size: 12px;
+}
+
+body {
+  margin: 0;
+  padding: 0;
+}
+
+ul {
+  padding: 0;
+  margin: 0;
+}
+
+.header {
+  background-color: #fff;
+  display: flex;
+  padding-left: 50px;
+  padding-right: 50px;
+  justify-content: space-between;
+  box-shadow: 0 2px 4px rgba(0, 0, 0, 0.05);
+  align-items: center;
+}
+
+.logo {
+  width: 40px;
+  height: 40px;
+}
+
+.nav--list {
+  display: flex;
+  margin: 0;
+  flex-wrap: wrap;
+  list-style-type: none;
+}
+
+.nav--item:nth-child(n+2) {
+  margin-left: 20px;
+}
+
+.nav--link:link {
```

Comment:
так стилізуються лише не відвідані посилання
<img width="263" alt="image" src="https://user-images.githubusercontent.com/66749919/188264326-19913cfb-a91a-4aa1-b889-9f90ea4a21d0.png">

```suggestion
.nav--link {
```